docs(firestore-bigquery-export): collectionGroup import script documentation improvement (#535)
firebase/extensions: firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md

  - If document changes occur in the time between installing the extension and running the import script.
  - If you run the import script multiple times over the same collection.

> - You cannot use wildcard notation in the collection path (i.e. `/collection/{document}/sub_collection}`). It is possible to perform a [collectionGroup](https://firebase.google.com/docs/firestore/query-data/queries#collection-group-query) query.
> 
> To use a `collectionGroup` query, provide the collection name value as `${COLLECTION_PATH}`, and set `${COLLECTION_GROUP_QUERY}` value to `true`. Warning: A `collectionGroup` query will target every collection in your Firestore project with the provided `${COLLECTION_PATH}`. For example, if you have 10,000 documents with a sub-collection named: `landmarks`, the import script will query every document in 10,000 `landmarks` collections.

Reviewer: ```suggestion
- You cannot use wildcard notation in the collection path (i.e. `/collection/{document}/sub_collection}`). Instead, you can use a [collectionGroup](https://firebase.google.com/docs/firestore/query-data/queries#collection-group-query) query. To use a `collectionGroup` query, provide the collection name value as `${COLLECTION_PATH}`, and set `${COLLECTION_GROUP_QUERY}` to `true`.
  
  Warning: A `collectionGroup` query will target every collection in your Firestore project with the provided `${COLLECTION_PATH}`. For example, if you have 10,000 documents with a sub-collection named: `landmarks`, the import script will query every document in 10,000 `landmarks` collections.
```
